controller_manager/test/test_spawner_unspawner.cpp

```diff
@@ -156,6 +156,93 @@ TEST_F(TestLoadController, spawner_test_type_in_param)
   }
 }
 
+TEST_F(TestLoadController, multi_ctrls_test_type_in_param)
+{
+  cm_->set_parameter(rclcpp::Parameter("ctrl_1.type", test_controller::TEST_CONTROLLER_CLASS_NAME));
+  cm_->set_parameter(rclcpp::Parameter("ctrl_2.type", test_controller::TEST_CONTROLLER_CLASS_NAME));
+  cm_->set_parameter(rclcpp::Parameter("ctrl_3.type", test_controller::TEST_CONTROLLER_CLASS_NAME));
+
+  ControllerManagerRunner cm_runner(this);
+  EXPECT_EQ(call_spawner("ctrl_1 ctrl_2 -c test_controller_manager"), 0);
```

Comment:
```suggestion
  EXPECT_EQ(call_spawner("--controller-names ctrl_1 ctrl_2 -c test_controller_manager"), 0);
```
it'd be nice to also have one that says 

```
EXPECT_EQ(call_spawner("--controller-name ctrl_1 -c test_controller_manager"), 0);
```